src/registrar/templates/application_other_contacts.html

```diff
@@ -74,6 +88,9 @@ <h2>Organization contact {{ forloop.counter }} (optional)</h2>
           <legend>
             <h2>No other employees from your organization?</h2>
```

Comment:
@rachidatecs can we remove the bottom margin here of 8px. There is already 16ishpx of margin above the body below this question and it feels too spaced out.

Reply:
I can't seem to find it in this file, but can you change `<fieldset class="usa-fieldset margin-top-2">` to have a top margin of rem 2?

I don't know if I'm referring to it correctly, but looking at this `_utility-builder.scss` in the styles panel it is showing 1 rem.

Reply:
In human words, the margin above "No other employees from your organization"